scriptmodules/libretrocores/lr-mame2003.sh

```diff
@@ -59,17 +59,19 @@ function configure_lr-mame2003() {
         for mame_sub_dir in cfg ctrlr diff hi memcard nvram; do
             mkRomDir "$mame_dir/$dir_name/$mame_sub_dir"
         done
-
-        # lr-mame2003-plus also has an artwork folder
-        [[ "$md_id" == "lr-mame2003-plus" ]] && mkRomDir "$mame_dir/$dir_name/artwork"
     done
 
     mkUserDir "$biosdir/$dir_name"
     mkUserDir "$biosdir/$dir_name/samples"
-
-    # copy hiscore.dat
+    
+    # copy hiscore.dat and cheat.dat
     cp "$md_inst/metadata/"{hiscore.dat,cheat.dat} "$biosdir/$dir_name/"
     chown $user:$user "$biosdir/$dir_name/"{hiscore.dat,cheat.dat}
+    
+    # lr-mame2003-plus also has an artwork folder
+    [[ "$md_id" == "lr-mame2003-plus" ]] && mkRomDir "$biosdir/$dir_name/artwork"
+    [[ "$md_id" == "lr-mame2003-plus" ]] && cp "$md_inst/metadata/artwork/*.*" "$biosdir/$dir_name/artwork/"
+    [[ "$md_id" == "lr-mame2003-plus" ]] && chown $user:$user "$biosdir/$dir_name/artwork/*.*"
```

Comment:
An `if` block would be better for these lines.

Reply:
Thank you -- I've added an `if` block with the latest commit.